cheroot/test/conftest.py

```diff
@@ -12,21 +12,40 @@
 
 import pytest
 
+import cheroot
+
 from ..server import Gateway, HTTPServer
-from ..testing import (  # noqa: F401
-    native_server, wsgi_server,
+from ..testing import (
+    check_log_messages,
+    cheroot_server,
+    get_server_client,
 )
-from ..testing import get_server_client
 
 
 @pytest.fixture
-def wsgi_server_client(wsgi_server):  # noqa: F811
+def wsgi_server(caplog):
```

Comment:
these were both moved from `testing.py` in order to resolve the `F811` lint error. they were previously `scope=module` - i removed that in order to ensure that they're brought up and torn down cleanly for each test.

Reply:
My intention was to have things that would later become official testing helpers in `cheroot.testing`. That's why I kept them there. Of course, the fixtures will need refactoring before they are truly ready for public consumption (the generator based interfaces are not extendable, nor composable).
So I'm not sure if moving them around is a good thing.